autoload/vaffle.vim

```diff
@@ -379,4 +379,21 @@ function! vaffle#fill_cmdline() abort
 endfunction
 
 
+function! vaffle#chdir(...) abort
```

Comment:
I concern about the difference of the behavior against `cd` command.
- `cd` without arguments indicates **user's home directory**
- but `vaffle#chdir()` without arguments indicates **the current working directory**

So I think `vaffle#chdir()` should be `vaffle#chdir(path)` and add another function like `vaffle#chdir_here()`. What do you think?

Reply:
I got it: analogy with `cd`